packages/cli/src/commands/init.ts

```diff
@@ -221,6 +241,53 @@ export default class Init extends Command {
       cli.action.stop();
     }
     this.log(`${this.project.name} is ready`);
-    process.exit(0);
+  }
+  async createProjectScaffold(): Promise<void> {
+    await prepareProjectScaffold(this.projectPath);
+
+    const {abiFilePath} = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'abiFilePath',
+        message: 'Path to ABI',
+        validate(input: string): boolean | string | Promise<boolean | string> {
+          if (!path.isAbsolute(path.resolve(input))) {
+            return 'Please enter an absolute file path';
+          }
+          return true;
+        },
+      },
+    ]);
+
+    const {contractAddress} = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'contractAddress',
+        message: 'Please provide an address (optional)',
+        default: '',
+      },
+    ]);
+
+    const {startBlock} = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'startBlock',
+        message: 'Please provide startBlock (optional)',
```

Comment:
I don't think this can be optional
```suggestion
        message: 'Please provide startBlock when the contract was deployed or first used',
```